src/main/java/io/seqera/tower/cli/responses/RunView.java

```diff
@@ -48,92 +133,109 @@ public void toString(PrintWriter out) {
-                manifestTable.addRow("NF version", workflow.getManifest().getNextflowVersion());
-                ++manifestCount;
-            }
-            if (workflow.getManifest().getDefaultBranch() != null) {
-                manifestTable.addRow("Default branch", workflow.getManifest().getDefaultBranch());
-                ++manifestCount;
-            }
-            if (workflow.getManifest().getVersion() != null) {
-                manifestTable.addRow("Version", workflow.getManifest().getVersion());
-                ++manifestCount;
-            }
-            if (workflow.getManifest().getHomePage() != null) {
-                manifestTable.addRow("Homepage", workflow.getManifest().getHomePage());
-                ++manifestCount;
-            }
-            if (workflow.getManifest().getGitmodules() != null) {
-                manifestTable.addRow("Git modules", workflow.getManifest().getGitmodules());
-                ++manifestCount;
-            }
-            if (workflow.getManifest().getDescription() != null) {
-                manifestTable.addRow("Description", workflow.getManifest().getDescription());
-                ++manifestCount;
-            }
-            if (workflow.getManifest().getName() != null) {
-                manifestTable.addRow("Name", workflow.getManifest().getName());
-                ++manifestCount;
-            }
-            if (workflow.getManifest().getMainScript() != null) {
-                manifestTable.addRow("Main script", workflow.getManifest().getMainScript());
-                ++manifestCount;
-            }
-            if (workflow.getManifest().getAuthor() != null) {
-                manifestTable.addRow("Author", workflow.getManifest().getAuthor());
-                ++manifestCount;
-            }
-
-            if (manifestCount > 0) {
-                out.println(ansi(String.format("%n%n    @|bold Manifest|@")));
-                manifestTable.print();
-            }
-        }
-
-        if (workflow.getStats() != null) {
-            out.println(ansi(String.format("%n%n    @|bold Status|@")));
-
-            TableList statsTable = new TableList(out, 2);
-            statsTable.setPrefix("    ");
-            statsTable.addRow("Pending", workflowLoad.getPending().toString());
-            statsTable.addRow("Submitted", workflowLoad.getSubmitted().toString());
-            statsTable.addRow("Running", workflowLoad.getRunning().toString());
-            statsTable.addRow("Cached", workflowLoad.getCached().toString());
-            statsTable.addRow("Succeeded", workflowLoad.getSucceeded().toString());
-            statsTable.addRow("Failed", workflowLoad.getFailed().toString());
-            statsTable.print();
-        }
-
-
-        if (workflow.getParams() != null && !workflow.getParams().isEmpty()) {
+        if (!config.isEmpty()) {
+            out.println(ansi(String.format("%n    @|bold Configuration|@")));
+            out.println(ansi(config.toString()));
```

Comment:
This can be improved by formating the map item by item, but it is time-consuming, and thought it was not a priority at this moment. Perhaps we can work it out in future releases.